Mathlib/LinearAlgebra/Matrix/Permutation.lean

```diff
@@ -60,4 +61,38 @@ theorem trace_permutation [AddCommMonoidWithOne R] :
   delta trace
   simp [toPEquiv_apply, ← Set.ncard_coe_finset, Function.fixedPoints, Function.IsFixedPt]
 
+lemma permMatrix_mulVec {v : n → R} [CommRing R] :
```

Comment:
@b-mehta Separate note not for this PR: I feel the name `Equiv.Perm.permMatrix` is pretty redundant. I think we could probably rename as `Equiv.Perm.matrix` or `Equiv.Perm.toMatrix`.